app/static/ringing_room.js

```diff
@@ -1095,6 +1104,12 @@ $(document).ready(function() {
                     >
                 [[ assignment_mode ? 'Stop assigning' : 'Assign bells' ]]
             </button>
+            <button class="btn btn-outline-primary w-100"
```

Comment:
Adjust vertical spacing slightly:

```suggestion
            <button class="btn btn-outline-primary w-100 mt-2 mb-n1"
```